src/platforms/javascript/common/performance.mdx

```diff
@@ -1,91 +1,58 @@
-To get started with performance monitoring using Sentry's JavaScript SDK, first install the `@sentry/browser` and `@sentry/tracing` packages:
+---
+title: Performance
+description: "Learn more about how to configure our Performance integrations to get the best experience out of it."
+---
 
-```bash
-# Using yarn
-$ yarn add @sentry/browser @sentry/tracing
+<Alert level="info" title="Note"><markdown>
 
-# Using npm
-$ npm install --save @sentry/browser @sentry/tracing
-```
+The `BrowserTracing` package is designed to work with `@sentry/tracing` and one of our browser JavaScript packages (`@sentry/browser`, `@sentry/react`, `@sentry/angular` etc.) and requires those packages to be installed and set up. For information on how to set up performance monitoring with JavaScript, see [the Performance documentation](../../#monitor-performance)
 
-<Alert level="warning" title="Note"><markdown>
+</markdown></Alert>
 
-We switched our package name from `@sentry/apm` to `@sentry/tracing`. If you were previously using `@sentry/apm`, please see [Switching to @sentry/tracing](/product/performance/apm-to-tracing/).
+### Automatic Instrumentation
 
-</markdown></Alert>
+The `BrowserTracing` integration provides automatic instrumentation to monitor the performance of browser applications.
 
-Next, initialize the integration in your call to `Sentry.init`:
+The `BrowserTracing` integration resides in the `@sentry/tracing` package. You can add it to your `Sentry.init` call:
 
-```jsx
+```javascript {tabTitle: ESM}
 import * as Sentry from "@sentry/browser";
 import { Integrations } from "@sentry/tracing";
 Sentry.init({
   dsn: "___PUBLIC_DSN___",
-  release: "my-project-name@" + process.env.npm_package_version,
   integrations: [new Integrations.BrowserTracing()],
   tracesSampleRate: 1.0, // Be sure to lower this in production
 });
 ```
 
-Alternatively, instead of npm packages, you can use our pre-built CDN bundle that combines both `@sentry/browser` and `@sentry/tracing`:
-
-<JsCdnTag tracing />
-
-Next, initialize the integration in your call to `Sentry.init`:
-
-```js
+```javascript {tabTitle: CDN}
```

Comment:
In this PR (https://github.com/getsentry/sentry-docs/pull/2160) , the CDN pieces moved into their own page (https://docs.sentry.io/platforms/javascript/configuration/install-cdn) based on feedback on the JS Getting Started content.